x/escrow/keeper/keeper_settle_test.go

```
@@ -100,14 +101,42 @@
 		},
 	} {
 		account, payments, blocks := setupDistTest(t, tt.cfg)
+		//initialBalance := sdkmath.LegacyMustNewDecFromStr(account.State.Deposits[0].Balance.Amount.String())
```

golangci-lint (2.5.0)

[error] commentFormatting: put a space between `//` and comment text

(gocritic)